ITRFConverter.h

```diff
@@ -0,0 +1,51 @@
+//# Dir2ITRF.h: Convertor that maps time to an ITRF direction.
+
+#ifndef LOFAR_STATIONRESPONSE_DIR2ITRF_H
+#define LOFAR_STATIONRESPONSE_DIR2ITRF_H
+
+// \file
+// Functor that maps J2000 to an ITRF direction.
+
+#include "Types.h"
+
+#include <casacore/measures/Measures/MeasFrame.h>
+#include <casacore/measures/Measures/MeasConvert.h>
+#include <casacore/measures/Measures/MCDirection.h>
+
+#include <memory>
+
+namespace LOFAR
+{
+namespace StationResponse
+{
+
+// \addtogroup StationResponse
+// @{
+
+class ITRFConverter
```

Comment:
At some point it would be nice to have comments in this class too, but that can wait.